sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -62,17 +63,44 @@ def evaluate(
 
     Examples
     --------
+        The type of evaluation that is done by `evaluate` depends on metrics in
+        param `scoring`
+        When evaluating model/estimators on point forecast, users can either let
+        scoring=None, which defaults to MeanAbsolutePercentageError
     >>> from sktime.datasets import load_airline
     >>> from sktime.forecasting.model_evaluation import evaluate
     >>> from sktime.forecasting.model_selection import ExpandingWindowSplitter
     >>> from sktime.forecasting.naive import NaiveForecaster
     >>> y = load_airline()
     >>> forecaster = NaiveForecaster(strategy="mean", sp=12)
-    >>> cv = ExpandingWindowSplitter(
-    ...     initial_window=24,
-    ...     step_length=12,
-    ...     fh=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12])
+    >>> cv = ExpandingWindowSplitter(initial_window=12, step_length=3, fh=[1, 2, 3])
     >>> results = evaluate(forecaster=forecaster, y=y, cv=cv)
+
+        Optionally, users may select other metrics that are compatible
```

Comment:
replace "comaptible with point for cast" better with "metrics can be supplied by the `scoring` argument. These can be forecast metrics of any kind, i.e., point forecast metrics, interval metrics, quantile foreast metrics.

Reply:
Done